apps/api/src/hyp3_api/handlers.py

```diff
@@ -82,9 +85,33 @@ def post_jobs(body, user):
     return body
 
 
-def get_jobs(user, start=None, end=None, status_code=None, name=None):
-    jobs = dynamo.query_jobs(user, start, end, status_code, name)
-    return {'jobs': jobs}
+def build_next_token(next_token):
```

Comment:
`build_next_token` seems like a weird name when you're passing in `next_token`, maybe `encode_token` or `encode_next_token`